tests/queries/tests.py

```diff
@@ -2523,32 +2523,34 @@ def test_null_ordering_added(self):
         self.assertEqual(sql.find(fragment, pos + 1), -1)
         self.assertEqual(sql.find("NULL", pos + len(fragment)), pos + len(fragment))
 
-    # Sqlite 3 does not support passing in more than 1000 parameters except by
-    # changing a parameter at compilation time.
-    @skipUnlessDBFeature('supports_1000_query_parameters')
     def test_ticket14244(self):
         # The "in" lookup works with lists of 1000 items or more.
         # The numbers amount is picked to force three different IN batches
         # for Oracle, yet to be less than 2100 parameter limit for MSSQL.
         numbers = list(range(2050))
         Number.objects.all().delete()
```

Comment:
Not sure why this line is required, we don't seem to be creating any `Number` in `setupTestData()` 🤷‍♂️